README.md

```diff
@@ -108,3 +108,20 @@ INFO[2021-09-17T05:44:04.317992888Z] converted image 192.168.1.1/library/nginx:l
 ```
 
 After that, we will find converted artifact in Harbor interface with name `<harbor-service-address>/nginx:latest-nydus`.
+
+### Daemonless mode
+
+Daemonless mode allows to do a one-time conversion without starting the `acceld` service, using `accelctl` like this:
+
+Note: only requires a local containerd service.
+
```

Comment:
"Daemonless" sounds a little bit confusing because it still requires containerd daemon. Something like "one-time mode" or "manual conversion" might be better.

Reply:
Fixed, changed to one-time mode.